.circleci/config.yml

```diff
@@ -5,15 +5,21 @@ jobs:
     environment:
       JAVA_TOOL_OPTIONS: -Xms512m -Xmx2g
     docker:
-      - image: circleci/openjdk:8u171-jdk
+      - image: adoptopenjdk/openjdk8:jdk8u202-b08
```

Comment:
I pinned the openjdk image previously due to a bug (https://github.com/dropwizard/dropwizard/pull/2559). Do we still want to pin to a specific version? Else we can just use latest

Reply:
@nickbabcock I was playing around with different images and also could reproduce the failing builds with CircleCI's OpenJDK 8u181 image.

This being said, the OpenJDK 11 image seems to work, so I'd go with this one in the future.